KubeArmor/eventAuditor/BPF/Makefile

```diff
@@ -0,0 +1,92 @@
+ifeq ($(V), 1)
```

Comment:
If this Makefile is inspired by some existing Makefile/repo, please ensure we attribute that repo/project in the header. For e.g, 
```
This makefile is inspired by libbpf/bootstrap.
```
If there is license attribution needed, please add that too.

Reply:
@nyrahul I don't really know how to determine the composition and creation of this Makefile. Let's see. Here we are using a libbpf submodule that needs to be compiled beforehand, so the Makefile will require rules for that. For the correct use of libbpf we'll need vmlinux.h, so we have to use bpftool and create some rules to create it and check the btf kernel file. Later, we'll need to compile the bpf objects that also require other rules. 

Has libbpf-bootstrap some similar steps? Yes it has (and it does have license).

https://github.com/libbpf/libbpf-bootstrap/blob/master/examples/c/Makefile

Has libbpfgo some similar steps? Yes it has (and it doesn't have license).

https://github.com/aquasecurity/libbpfgo/blob/main/Makefile

Has linux/tools/testing/selftests/bpf some similar steps? Yes it has (and it does have license).

https://github.com/torvalds/linux/blob/master/tools/testing/selftests/bpf/Makefile

I suppose that the steps **needed** to build the solution are not attached to the solution license, but let me know. Thanks.

Reply:
The logic is plain and simple. If you did copy-paste verbatim from some existing Makefile then it is always best to attribute the source (whichever source you picked up from). It doesn't hurt to say, this Makefile was inspired from so and so Makefile.

Reply:
> It doesn't hurt to say, this Makefile was inspired from so and so Makefile.

Exactly. I totally agree. I was just not certain since this file was not copied-paste verbatim, but inspired from various Makefiles. However, as you suggested I'll include the information about the inspirations and the related licenses.

Reply:
Done.